src/cljs/athens/devcards/all_pages.cljs

```diff
@@ -51,37 +55,82 @@
     (.toLocaleString  (js/Date. x))))
 
 
+(def table-cell-background-color-hover
+  (opacify (:panel-color HSL-COLORS) (first OPACITIES)))
+
+
+(def tables
+  {:width "100%"
+   :text-align "left"
+   :border-collapse "collapse"
+   ::stylefy/sub-styles {:thead {}
+                         :tr-item {}
+                         :th-title {}
+                         :th-body {}
+                         :th-date {:text-align "right"}
+                         :td-title {:width "15vw"
+                                    :min-width "10em"
+                                    :word-break "break-word"
+                                    :font-weight "500"
+                                    :font-size "21px"
+                                    :line-height "27px"}
+                         :td-body {}
+                         :body-preview {:white-space "wrap"
+                                        :word-break "break-word"
+                                        :overflow "hidden"
+                                        :text-overflow "ellipsis"
+                                        :display "-webkit-box"
+                                        :-webkit-line-clamp "3"
+                                        :-webkit-box-orient "vertical"}
+                         :td-date {:text-align "right"
+                                   :opacity "0.75"
+                                   :font-size "12px"
+                                   :min-width "9em"}}
+   ::stylefy/manual [[:tbody {:vertical-align "top"}
+                      [:tr
+                       [:td {:border-top (str "1px solid " (:panel-color COLORS))}]
+                       [:&:hover {:background-color table-cell-background-color-hover
+                                  :border-radius "8px"}
+                        ;; [:td {:border-top-color "transparent"}]
+                        [:td [(selectors/& (selectors/first-child)) {:border-radius "8px 0 0 8px"
+                                                                     :box-shadow "-16px 0 hsla(30, 11.11%, 93%, 0.1)"}]]
+                        [:td [(selectors/& (selectors/last-child)) {:border-radius "0 8px 8px 0"
+                                                                    :box-shadow "16px 0 hsla(30, 11.11%, 93%, 0.1)"}]]]]]
```

Comment:
What do these do?

Reply:
@tangjeff0 These add a little bit of extra background to the hovered items. Tables don't allow for this kind of thing without either crazy-bad markup or a little bit of CSS trickery.

Reference (the left edge of the table ends where the border does):
![Screen Shot 2020-06-11 at 9 09 05 PM](https://user-images.githubusercontent.com/98312/84454222-f279ec00-ac27-11ea-9832-901e01fe5ac5.png)